src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -2,11 +2,15 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Optional;
 
 import seedu.address.commons.core.index.Index;
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.attendance.Week;
 import seedu.address.model.person.Address;
```

Comment:
Not urgent, but perhaps we should also start removing all references to Address, Tag etc. - just remembered this

Reply:
Could maybe do it as a PR in v1.3b 💀

Reply:
Yeap thats true, will add issues so that we don't forget